pmc/README.md

```diff
@@ -0,0 +1,6 @@
+# Directory Structure
+
+| Folder/file | Description |
```

Comment:
Similar comment as before, if there is an order to running the notebooks that might be helpful to add here